docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -332,6 +349,26 @@ Then context element is in focus
 When I press ENTER on keyboard
 ----
 
+==== Check the element focus state
+
+Checks if the element found by the specified locator is in the provided focus state by comparing the found element and
+the active element
+
+[source,gherkin]
+----
+Then element located by `$locator` is $focusState
+----
+* `$locator` - The <<_locator,locator>> to find an element.
+* `$focusState` - The state to verify: `in focus` or `not in focus`.
+
+.Check the element with ID "input-username" is in focus and do keyboard input
+[source,gherkin]
+----
+Then element located by `id(input-username)` is $focusState
```

Comment:
$focusState -> focus?

Reply:
okay, I just thought we use real values in examples, not parameter refs

Reply:
oh, my bad, I didn't realized this is an example